src/main/java/org/smartregister/pathevaluator/plan/PlanEvaluator.java

```diff
@@ -88,7 +96,7 @@ private void evaluatePlan(PlanDefinition planDefinition, TriggerType triggerEven
 					evaluatePlan(planDefinition, triggerEvent, jurisdiction, null);
 					List<String> locationIds = locationDao.findChildLocationByJurisdiction(jurisdiction.getCode());
 					for (String locationId : locationIds) {
-						evaluatePlan(planDefinition, triggerEvent, new Jurisdiction(locationId), null);
+						queuingHelper.addToQueue(planDefinition.getIdentifier(),triggerEvent,locationId);
```

Comment:
Have we intentionally replaced evaluatePlan(planDefinition, triggerEvent, new Jurisdiction(locationId), null); with insertion to queue...